packages/dart/noports_core/lib/src/sshnp/util/srvd_channel/srvd_channel.dart

```diff
@@ -106,8 +103,8 @@ abstract class SrvdChannel<T> with AsyncInitialization, AtClientBindings {
     if (directSsh) {
       srv = srvGenerator(
         host,
-        _portA!,
-        localPort: localRvPort!,
+        clientPort,
+        localPort: localRvPort,
```

Comment:
using clientPort rather than _portA for readability
removed unsafe non-null  assertion